dali/core/mm/malloc_resource.cc

```diff
@@ -94,9 +94,8 @@ cuda_malloc_async_memory_resource::cuda_malloc_async_memory_resource(int device_
   dummy_host_stream_ = CUDAStreamPool::instance().Get(device_id_);
 #if NVML_ENABLED
   static const float driverVersion = []() {
-    nvml::Init();
+    auto nvml_handle = nvml::NvmlInstance::CreateNvmlHandle();
```

Comment:
```suggestion
    auto nvml_handle = nvml::NvmlInstance::CreateNvmlInstance();
```
Not full search and replace? :)

Reply:
Missed that. Fixed.